providers/AuthProvider.js

```diff
@@ -71,7 +71,12 @@ function AuthProvider(props = {}) {
       } else {
         if (!authenticated) {
           window.localStorage.removeItem(AUTH_TOKEN_KEY);
-          router.push('/');
+
```

Comment:
@rkdavidson looks like there is a conflict with this and the other PR I just approved and merged. Do we need this here?

Reply:
Yeah was afraid of that, got my PRs mixed up a bit with the churn around this logout behavior.

I it's not needed, the router line and import/usage in this file can be removed. I can deal with this in a bit when I get home. Sorry about that.

Reply:
Fixed in 35fe10d